src/test/csi-neonsan/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+
+appVersion: 1.2.0
+name: csi-neonsan
+description: A Helm chart for NeonSAN CSI Driver
+version: 1.2.0-rc1
+kubeVersion: ">=1.14.0-0"
```

Comment:
Does QingStor CSI support k8s v1.14?

Reply:
support.
1.14: volume management
1.15 1.16: volume management & expanding
support volume management,  but with volume expanding
1.17:  volume management & expanding &snapshot

Reply:
If you want to support Kubernetes v1.14+, you must test it.

Reply:
yes ，i has tested it，and it's OK